mmcv/cnn/utils/weight_init.py

```diff
@@ -93,6 +93,18 @@ def bias_init_with_prob(prior_prob):
     return bias_init
 
 
+def init_log(init_name, layer_name, module_name, weight_shape, bias_shape,
+             bias_value):
+    print_log(
+        f'    initialize weights ({weight_shape}) of {layer_name} '
+        f'in {module_name} in with {init_name}',
+        logger='mmcv')
```

Comment:
We may not force the logger to be in mmcv because downstream projects have different logger name and we wish these can all be recorded in the log file.